Source/settings/Settings.js

```diff
@@ -3,9 +3,8 @@ import { isWrappedObject, getDocumentData } from '../dom/utils'
 
 function getPluginIdentifier() {
   if (!__command.pluginBundle()) {
-    throw new Error(
-      'It seems that the command is not running in a plugin. Bundle your command in a plugin to use the Settings API.'
-    )
+    // if we run a script from the Run Script panel, it won't have a bundle
+    return 'com.sketchapp.temporary-script'
```

Comment:
I wonder if this change has any impact anywhere else? I can't think of any scenario in which something would be expecting an error, so this is probably safe, but I thought I'd ask

Reply:
Yeah that's only happening in the Run Script panel so I'd say it's safe to assume nobody care